### DISCUSSION TITLE
Unable to add the web part on SP 2019 enviroment

### DISCUSSION THREAD

**Discussion by PetronelaManiliuc** [microsoft-search/pnp-modern-search#2723] [Q&A]
Heloo!
I had tried to add the web part on my Sp enviroment and failed.
![Capture](https://user-images.githubusercontent.com/49759246/215485210-a75a7ca2-10d4-464d-bb80-865318a5090d.PNG)

**Comment by wobba** [comment]
SharePoint on-prem only support up to SPFx v1.4.1. See https://learn.microsoft.com/en-us/sharepoint/dev/spfx/sharepoint-2019-support 

Online is now on v1.16.1 and has features not supported on the on-prem version, so the current package will not work for you.

Also see https://github.com/microsoft-search/pnp-modern-search/issues/33

**Reply by PetronelaManiliuc** [reply]
Are any versions that could work?

**Reply by wobba** [reply]
See the thread I referenced and follow what they point to https://github.com/pnp/sp-dev-solutions/issues/355#issuecomment-725252440
